package.json

```diff
@@ -1021,7 +1021,7 @@
     }
   },
   "engines": {
-    "vscode": "^1.26.0"
+    "vscode": "^1.31.0"
```

Comment:
We can't change this due to https://github.com/Microsoft/vscode-docker/issues/900